cmd/rook/object/user.go

```diff
@@ -0,0 +1,331 @@
+	}
+
+	if len(args) == 1 {
+		return fmt.Errorf("Missing required argument Display Name")
+	}
+
+	if len(args) > 2 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	user := model.ObjectUser{UserID: args[0], DisplayName: &args[1]}
+
+	if cmd.Flags().Lookup("email").Changed {
+		user.Email = &emailFlag
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := createUser(c, user)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func createUser(c client.RookRestClient, user model.ObjectUser) (string, error) {
+	user, err := c.CreateObjectUser(user)
+
+	if client.IsHttpStatusCode(err, 422) {
+		restErr := err.(client.RookRestError)
+		return "", fmt.Errorf(string(restErr.Body))
+	}
+
+	if err != nil {
+		return "", fmt.Errorf("failed to create user: %+v", err)
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintf(w, "User ID:\t%s\n", user.UserID)
+	fmt.Fprintf(w, "Display Name:\t%s\n", *user.DisplayName)
+	fmt.Fprintf(w, "Email:\t%s\n", *user.Email)
+	fmt.Fprintf(w, "Access Key:\t%s\n", *user.AccessKey)
+	fmt.Fprintf(w, "Secret Key:\t%s\n", *user.SecretKey)
+
+	w.Flush()
+	return fmt.Sprintf("User Created\n\n%s", buffer.String()), nil
+}
+
+var userUpdateCmd = &cobra.Command{
+	Use:   "update [User ID]",
+	Short: "Updates an object store user",
+}
+
+func updateUserEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if len(args) == 0 {
+		return fmt.Errorf("Missing required argument User ID")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	user := model.ObjectUser{UserID: args[0]}
+
+	if cmd.Flags().Lookup("display-name").Changed {
+		user.DisplayName = &displayNameFlag
+	}
+	if cmd.Flags().Lookup("email").Changed {
+		user.Email = &emailFlag
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := updateUser(c, user)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func updateUser(c client.RookRestClient, user model.ObjectUser) (string, error) {
+	updatedUser, err := c.UpdateObjectUser(user)
+
+	if client.IsHttpNotFound(err) {
+		return "", fmt.Errorf("Unable to find user %s", user.UserID)
+	}
+
+	if err != nil {
+		return "", fmt.Errorf("failed to update user: %+v", err)
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintf(w, "User ID:\t%s\n", updatedUser.UserID)
+	fmt.Fprintf(w, "Display Name:\t%s\n", *updatedUser.DisplayName)
+	fmt.Fprintf(w, "Email:\t%s\n", *updatedUser.Email)
+	fmt.Fprintf(w, "Access Key:\t%s\n", *updatedUser.AccessKey)
+	fmt.Fprintf(w, "Secret Key:\t%s\n", *updatedUser.SecretKey)
+
+	w.Flush()
+	return fmt.Sprintf("User Created\n\n%s", buffer.String()), nil
```

Comment:
why not just print the string instead of returning it? looks like all you do is print it anyway